src/wp-login.php

```diff
@@ -90,7 +90,7 @@ function login_header( $title = 'Log In', $message = '', $wp_error = null ) {
 	<html <?php language_attributes(); ?>>
 	<head>
 	<meta http-equiv="Content-Type" content="<?php bloginfo( 'html_type' ); ?>; charset=<?php bloginfo( 'charset' ); ?>" />
-	<title><?php echo $login_title; ?></title>
+	<title><?php echo esc_attr( $login_title ); ?></title>
```

Comment:
`esc_attr()` doesn't seem right here. This is the contents of a tag, not the contents of the `title` attribute of a tag. Looking at [_wp_render_title_tag](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/general-template.php#L1287), this doesn't escape the title, so I don't think it's necessary here.

Reply:
Instead, can we use? `<title><?php echo esc_html( $login_title ); ?></title>`